File: src/main/java/me/kuku/yuq/service/GroupService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.GroupEntity;

import java.util.List;

@AutoBind
public interface GroupService {
    void save(GroupEntity groupEntity);
    GroupEntity findByGroup(Long group);
    List<GroupEntity> findByOnTimeAlarm(Boolean onTimeAlarm);
    List<GroupEntity> findAll();
    List<GroupEntity> findByLocMonitor(Boolean locMonitor);
}

File: src/main/kotlin/me/kuku/yuq/service/GroupService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.GroupEntity

@AutoBind
interface GroupService {

    fun save(groupEntity: GroupEntity)
    fun findByGroup(group: Long): GroupEntity?
    fun findByOnTimeAlarm(onTimeAlarm: Boolean): List<GroupEntity>
    fun findAll(): List<GroupEntity>
    fun findByLocMonitor(locMonitor: Boolean): List<GroupEntity>

}